package.json

```diff
@@ -38,7 +40,9 @@
     "image"
   ],
   "dependencies": {
-    "prop-types": "^15.5.8"
+    "prop-types": "^15.5.8",
+    "react": "^15.5.4",
```

Comment:
I think we should leave them as `peerDependencies` + `devDependencies`, or why did you change that?